src/Symfony/Component/Console/Tests/Fixtures/application_1.rst

```diff
@@ -0,0 +1,398 @@
+Console Tool
+############
+
+- `completion`_
+- `help`_
+- `list`_
+
+``completion``
+**************
+
+Dump the shell completion script
+
+Usage
+=====
+
+- ``completion [--debug] [--] [<shell>]``
+
+The completion command dumps the shell completion script required
+to use shell autocompletion (currently only bash completion is supported).
+
+Static installation
+-------------------
+
+Dump the script to a global completion file and restart your shell:
```

Comment:
I haven't checked if this works too, but I think that "literal blocks" require:

* Contents to be indented by 4 spaces (this works perfectly in your code)
* The line before the literal block must end with two colons `::` (<-- this is the missing piece here)

See https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks

Reply:
Indeed, the line before the literal block *must* end with `::`. See also the official reStructuredText markup specification: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#literal-blocks